app/webpack/reactjs/components/close_button.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from "react";
+
+interface CloseButtonProps {
+  handleClick: () => void;
+}
+
+export const CloseButton: FC<CloseButtonProps> = ({ handleClick }) => {
```

Comment:
This was in the credits modal and we use this thing all over the place.  So as we move to react, let's just have a common one.